packages/compass-web/scripts/electron-proxy.js

```diff
@@ -213,67 +242,116 @@ class AtlasCloudAuthenticator {
     })());
   }
 
-  async logout() {
-    await session.defaultSession.clearStorageData({ storages: ['cookies'] });
+  async cleanupAndLogout() {
+    // When logging out, delete the test user too. If we don't do this now, we
+    // will loose a chance to do it later due to missing auth
+    try {
+      await atlasCloudAuthenticator.deleteTestDBUser();
+    } catch {
+      // Can fail if user wasn't even created yet
```

Comment:
nit: maybe worth at least debug logging when we ignore errors just to make sure that it is the error we expect and not some NPE or other programmer error.